apps/api/v2/src/modules/cal-unified-calendars/outputs/get-unified-calendar-event.ts

```diff
@@ -0,0 +1,166 @@
+import { ApiProperty, ApiPropertyOptional } from "@nestjs/swagger";
+import { Type } from "class-transformer";
+import { IsEnum, IsISO8601, IsOptional, IsString, ValidateNested } from "class-validator";
+
+import { CALENDARS, SUCCESS_STATUS, ERROR_STATUS } from "@calcom/platform-constants";
+
+export class DateTimeWithZone {
+  @IsISO8601()
+  time!: string;
+
+  @IsString()
+  timeZone!: string;
+}
+
+export class UnifiedCalendarEventOutput {
+  @ValidateNested()
+  @Type(() => DateTimeWithZone)
+  @ApiProperty({
+    type: "object",
+    properties: {
+      time: { type: "string", format: "date-time" },
+      timeZone: { type: "string" },
+    },
+    description: "Start date and time of the calendar event with timezone information",
+  })
+  start!: DateTimeWithZone;
+
+  @ValidateNested()
+  @Type(() => DateTimeWithZone)
+  @ApiProperty({
+    type: "object",
+    properties: {
+      time: { type: "string", format: "date-time" },
+      timeZone: { type: "string" },
+    },
+    description: "End date and time of the calendar event with timezone information",
+  })
+  end!: DateTimeWithZone;
+
+  @IsString()
+  @ApiProperty({
+    type: String,
+    description: "Unique identifier of the calendar event",
+  })
+  id!: string;
+
+  @IsString()
+  @ApiProperty({
+    type: String,
+    description: "Title of the calendar event",
+  })
+  title!: string;
+
+  @IsString()
+  @IsOptional()
+  @ApiPropertyOptional({
+    type: String,
+    nullable: true,
+    description: "Detailed description of the calendar event",
+  })
+  description?: string | null;
+
+  @IsString()
+  @IsOptional()
+  @ApiPropertyOptional({
+    type: String,
+    nullable: true,
+    description: "Location of the calendar event",
+  })
+  location?: string | null;
+
+  @IsOptional()
+  @ApiPropertyOptional({
+    type: "array",
+    items: {
+      type: "object",
+      properties: {
+        entryPointType: { type: "string" },
+        uri: { type: "string" },
+        label: { type: "string" },
+        pin: { type: "string", nullable: true },
+        regionCode: { type: "string", nullable: true },
+      },
+    },
+    nullable: true,
+    description: "Conference locations with entry points (video, phone, etc.)",
+  })
+  locations?: Array<{
+    entryPointType: string;
+    uri: string;
+    label?: string;
+    pin?: string;
+    regionCode?: string;
+  }> | null;
+
+  @IsOptional()
+  @ApiPropertyOptional({
+    type: "array",
+    items: {
+      type: "object",
+      properties: {
+        email: { type: "string" },
+        name: { type: "string" },
+        responseStatus: { type: "string", nullable: true },
+        organizer: { type: "boolean", nullable: true },
+        self: { type: "boolean", nullable: true },
```

Comment:
Can we have `description` field added to each property and same for locations and hosts so that people know what is "responseStatus", what is "self" etc.